packages/types/api.d.ts

```diff
@@ -2119,6 +2127,147 @@ export interface IParserErrorMessageProvider {
   }): string
 }
 
+export declare class LLkLookaheadStrategy implements ILookaheadStrategy {
+  validateNoLeftRecursion(rules: Rule[]): ILookaheadValidationError[]
```

Comment:
Yeah, I was confused at first as well, but TypeScript requires `declare class` to explicitly declare all implemented methods of an interface:

![image](https://user-images.githubusercontent.com/4377073/192139018-7e3aaa65-9bff-42f2-a70c-5e748a2cec87.png)

> Could a constructor interface resolve the duplicated signatures?

I don't think so. I believe we would need a factory to solve this, but it'd be great if Chevrotain would explicitly export its own lookahead strategy, so that other strategies can build on top of that. This is also why the validation methods are declared in the `api.d.ts`, since the LL-Star package would make use of that (they have a lot of overlapping, LL-specific validations).